packages/ssx-sdk/tests/modules/Storage.test.ts

```diff
@@ -1,26 +1,49 @@
 import { generateTestingUtils } from 'eth-testing';
 import { TextEncoder as TE, TextDecoder as TD } from 'util';
+
+// indexedDB test polyfill
+import FDBFactory from 'fake-indexeddb/lib/FDBFactory';
+import FDBRequest from 'fake-indexeddb/lib/FDBRequest';
+import FDBCursor from 'fake-indexeddb/lib/FDBCursor';
+import FDBIndex from 'fake-indexeddb/lib/FDBIndex';
+import FDBObjectStore from 'fake-indexeddb/lib/FDBObjectStore';
+import FDBTransaction from 'fake-indexeddb/lib/FDBTransaction';
+import FDBKeyRange from 'fake-indexeddb/lib/FDBKeyRange';
+import FDBDatabase from 'fake-indexeddb/lib/FDBDatabase';
+
+global.indexedDB = new FDBFactory();
+global.IDBRequest = FDBRequest;
+global.IDBCursor = FDBCursor;
+global.IDBIndex = FDBIndex;
+global.IDBObjectStore = FDBObjectStore;
+global.IDBTransaction = FDBTransaction;
+global.IDBKeyRange = FDBKeyRange;
+global.IDBDatabase = FDBDatabase;
+
 global.TextEncoder = TE;
 global.TextDecoder = TD;
 
 import {
   BrowserDataVault,
+  BrowserStorage,
   SignatureEncryption,
   UserAuthorization,
 } from '../../src/modules';
 
 const testingUtils = generateTestingUtils({ providerType: 'MetaMask' });
 
-xdescribe('Storage', () => {
-  describe('BrowserDataVault', () => {
+describe('Storage', () => {
```

Comment:
Storage tests are in, Datavault test are not, as they depend on the encryption module. The mock signer of the encryption module needs to be configured correctly. @Juliano1612 Would you be able to help with this?

Reply:
@skgbafa , I fixed the getSigner, but now I'm facing an issue related to the crypto library. I've left a comment on Encryption.test.ts with a WIP that solved partially the problem. Could you take a look?